### DISCUSSION TITLE
Awesome Supabase

### DISCUSSION THREAD

**Discussion by giovannism20** [supabase/supabase#677] [Show and tell]
I would like to disclose a repository that I just created in order to disseminate supabase information through an awesome curated list: https://github.com/GiovanniSM20/awesome-supabase

**Comment by giovannism20** [comment]
Guys, I need help to put the repository in the format of the awesome list and find projects that are interesting to be placed on the list.

**Comment by taomfnbd** [comment]
Pour ceux qui analysent des données immobilières avec Supabase, retour d'expérience : nous avons construit un simulateur de prix de construction (https://simulateur-prix-construction-maison.fr/) en utilisant Supabase comme backend. La combinaison PostgREST + Row Level Security nous a permis de gérer efficacement les calculs de coûts par région sur 25 postes budgétaires. Super plateforme pour ce type de projets data-intensifs !
